tests/configs.py

```diff
@@ -1,2 +1,2 @@
-ABS_TOL = 1e-7
+ABS_TOL = 1.1920928955078125e-07  # 1e-7 for float of width 16
```

Comment:
it looks like `2. ** -23`

Reply:
Yes, it is...
```
>>> 2. ** -23 == 1.1920928955078125e-07
True
```

Reply:
I mean, it can be written in a shorter and more meaningful way

Reply:
I understand. I just feel frustrated for not seeing before ;)